pkg/state/rbac_role_test.go

```diff
@@ -105,7 +105,8 @@ func TestRBACRolesCollection_Get(t *testing.T) {
 	}
 	rbacRole1 := RBACRole{
 		RBACRole: kong.RBACRole{
-			ID: kong.String("foo-id"),
+			ID:   kong.String("foo-id"),
+			Name: kong.String("foo-name"),
```

Comment:
I have no idea why this is needed now. Without it rbac tests fail after the bump of the go version.

To me it looks like it should've been there, we have [another test that checks that a similar condition](https://github.com/Kong/go-database-reconciler/blob/e72f4c2de6609998d5e5671e14a9f7c0cd6a4c3e/pkg/state/rbac_role_test.go#L36-L42) (without a name) fails. If anyone can explain this I'd be grateful.